### ISSUE TITLE
System DB gets locked.

### ISSUE THREAD

**Issue by Taz-Wolf** [EDDiscovery/EDDiscovery#3461]
I tabbed out of elite and saw the popup at the end of my windows list. ~~EDD is still running fine it looks like~~ It closed when closing the pop-up. I honestly don't know what the issue is. Here is the thingy it gave me to post here I assume:

```
An unhandled fatal exception has occurred.

==== BEGIN ====
code = Busy (5), message = System.Data.SQLite.SQLiteException (0x800007AF): database is locked
database is locked
   at System.Data.SQLite.SQLite3.Prepare(SQLiteConnection cnn, String strSql, SQLiteStatement previous, UInt32 timeoutMS, String& strRemain)
   at System.Data.SQLite.SQLiteCommand.BuildNextCommand()
   at System.Data.SQLite.SQLiteDataReader.NextResult()
   at System.Data.SQLite.SQLiteDataReader..ctor(SQLiteCommand cmd, CommandBehavior behave)
   at System.Data.SQLite.SQLiteCommand.ExecuteReader(CommandBehavior behavior)
   at System.Data.SQLite.SQLiteCommand.ExecuteNonQuery(CommandBehavior behavior)
   at System.Data.SQLite.SQLiteConnection.Open()
   at SQLLiteExtensions.SQLExtConnection..ctor(String dbfile, Boolean utctimeindicator, AccessMode mode, JournalModes journalmode, Boolean disallow_xthreading) in C:\Code\EDDiscovery\BaseUtilities\SQLLite\SQLLite\SQLConnection.cs:line 67
   at EliteDangerousCore.DB.SystemsDatabase.CreateConnection() in C:\Code\EDDiscovery\EliteDangerousCore\EliteDangerous\SystemDB\SystemsDatabase.cs:line 56
   at SQLLiteExtensions.SQLAdvProcessingThread`1.SqlThreadProc() in C:\Code\EDDiscovery\BaseUtilities\SQLLite\SQLLite\SQLAdvProcessingThread.cs:line 191
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.ThreadHelper.ThreadStart()
===== END =====
```

**Comment by Eahlstan** [comment]
Something is locking your system DB. Usually some kind of overprotective security app. Nothing we can do about that, in 99% this goes away on its own.

**Comment by klightspeed** [comment]
I wonder if a `DefaultFlags_SQLiteConnection` environment variable is present?

**Comment by FSFan** [comment]
Getting this one myself.  Normally will correct itself with a reboot.

**Comment by robbyxp1** [comment]
An experimental fix will be in 17.0.2, out soon.  In the meanwhile, restart.

**Comment by robbyxp1** [comment]
17.0.2 is out, we wait to see if it will stop these
